velox/functions/sparksql/Register.cpp

```diff
@@ -110,6 +110,8 @@ static void workAroundRegistrationMacro(const std::string& prefix) {
 
   VELOX_REGISTER_VECTOR_FUNCTION(
       udf_map_allow_duplicates, prefix + "map_from_arrays");
+  VELOX_REGISTER_VECTOR_FUNCTION(
+      udf_map, prefix + "map_from_arrays_exception_on_duplicates");
```

Comment:
Would you document this function in https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/map.rst?plain=1#L29? Thanks.

Reply:
@WangGuangxin, 
Seems this function has already been registered with a different name?
https://github.com/facebookincubator/velox/blob/6b2403307eb036070633bad4afb0b894c8148eb8/velox/functions/sparksql/Register.cpp#L277